githubReleaseScript.js

```diff
@@ -0,0 +1,27 @@
+const axios = require('axios');
+const config = require('./.config/.config.js')
+const { PAT } = config;
+
+async function createRelease() {
```

Comment:
Can you write a test case for `createRelease()` similar to the one from https://github.com/mongoosejs/backend/blob/master/test/webhookGithubApp.test.js ? Create an `src/createRelease.js` file, move `createRelease()` there, and write a test for it.